docker-compose.yml

```
@@ -24,61 +25,66 @@ services:
       - RAILS_SERVE_STATIC_FILES=true
       - RAILS_ENV=development
       - SETTINGS__FEATURE_FLAGS__ALLOW_JSON_UPLOAD=true
-    image: 'suldlss/dlme:latest'
     ports:
       - "3000:3000"
     depends_on:
       - postgres
     # For running in dev
     volumes:
       - ".:/opt/"
-    command: bundle exec rails server -b 0.0.0.0
+    entrypoint: dev-entrypoint.sh
+    command: rails server -b 0.0.0.0
+
+  sidekiq:
+    depends_on:
+      - "postgres"
+      - "redis"
+    build:
+      context: ./
+      dockerfile: docker/Dockerfile.worker
+      args:
+        # For development, installs test and development
+        - BUNDLE_WITHOUT=nothing
+    command: sidekiq -C config/sidekiq.yml.erb
+    volumes:
+      - ".:/app"
+    env_file:
+      - ".env"
+    environment:
+      - SOLR_URL=http://solr:8983/solr/dlme
+      - RAILS_ENV=development
+
   postgres:
-    image: postgres
+    image: postgres:10.17
     ports:
       - "5432:5432"
     environment:
       - POSTGRES_PASSWORD=sekret
       - POSTGRES_DB=dlme
     volumes:
       - postgres-data:/var/lib/postgresql/data
+
   solr:
-    image: solr
+    image: solr:8.11.1
     ports:
       - "8983:8983"
     volumes:
-      - ./solr-data:/opt/solr/server/solr/mycores
+      - solr-data:/var/solr
       - ./solr/config:/solr-setup/conf
     entrypoint:
       - docker-entrypoint.sh
       - solr-precreate
       - dlme
       - /solr-setup/conf
+
   redis:
-    image: 'redis:4.0-alpine'
+    image: "redis:4.0-alpine"
```

AWS is currently using 5.x 🤷‍♂️ 
```suggestion
    image: "redis:5.0-alpine"
```